lib/urlwatch/html2txt.py

```diff
@@ -36,18 +36,23 @@
 logger = logging.getLogger(__name__)
 
 
-def html2text(data, method='lynx'):
+def html2text(data, method='lynx', options=dict()):
```

Comment:
This doesn't work as expected, you must use `options=None` here and then:

    if options is None:
        options = {}

See also: https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python